quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-integration-test/src/test/java/org/kie/kogito/quarkus/workflows/ForEachRestIT.java

```diff
@@ -41,4 +41,16 @@ void testForEachRest() {
                 .body("workflowdata.output", is(Arrays.asList(2, 3, 4, 5, 6)))
                 .body("workflowdata.response", nullValue());
     }
+
+    @Test
+    void testForEachSubflow() {
+        given()
+                .contentType(ContentType.JSON)
+                .accept(ContentType.JSON)
+                .body("{\"numbers\" : [1,2,3,4,5], \"constant\": 2}}").when()
```

Comment:
Isn't a `}` remaining here?

```suggestion
                .body("{\"numbers\" : [1,2,3,4,5], \"constant\": 2}").when()
```